backend/apps/profile_info/models.py

```diff
@@ -0,0 +1,7 @@
+from apps.users.models import BaseUser
+
+class BaseUserProfile(BaseUser):
+    class Meta:
+        proxy = True  
```

Comment:
I'm not sure what the purpose of this model is, if it doesn't add any fields then it can be deleted and use the BaseUser model instead